poetry.lock

```diff
@@ -101,13 +101,13 @@ redis = ["redis (>=2.10.5)"]
 
 [[package]]
 name = "certifi"
-version = "2024.12.14"
```

Comment:
just curious what caused the lock file changes?  I didnt see any new things in the pyproject.toml

Reply:
in the course of installing and uninstalling the alternate packages, other things got updated. these changes are just newer versions of packages that have been updated. seems fine to let them get updated as we go.